test/e2e/bindata/assets/01_namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
can we separate all the qe related assets into a separate folder?

Maybe something like bindata/assets/qe/*